src/java/org/apache/cassandra/locator/SimpleStrategy.java

```diff
@@ -73,17 +72,19 @@ public EndpointsForRange calculateNaturalReplicas(Token token, TokenMetadata met
         return replicas.build();
     }
 
+    @Override
     public ReplicationFactor getReplicationFactor()
     {
         return rf;
     }
 
-    private final static void validateOptionsInternal(Map<String, String> configOptions) throws ConfigurationException
+    private static void validateOptionsInternal(Map<String, String> configOptions) throws ConfigurationException
     {
         if (configOptions.get(REPLICATION_FACTOR) == null)
             throw new ConfigurationException("SimpleStrategy requires a replication_factor strategy option.");
     }
 
+    @Override
```

Comment:
The method `recognizedOptions` right below could also benefit from an `@Override` annotation.